migrations/2024-12-29-203717_add_post_keyword_block_table/up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE user_post_keyword_block (
```

Comment:
This table name doesn't look like any other one.

- Look at `local_user_language` for an example.
- It should be named `local_user_keyword_block` , as its a local user setting, not a person field that we need to worry about federation.
- Its columns should be `local_user_id, keyword` (in that order)
- I don't think we need to reference that this applies only to posts : `keyword_block` is enough. In the future we might want this to apply to comments and other things anyway.
- The performance of this scares me, and I don't think there's really a way we can make `lower(post.title) not similar to '%(keyword_1|keyword_2|...)%'` performant. But I'm still willing to merge it and hack on that later.

Reply:
Also, "REFERENCES person" becomes "REFERENCES local_user"